py-polars/polars/internals/frame.py

```diff
@@ -2140,8 +2140,6 @@ def drop_nulls(self, subset: Optional[List[str]] = None) -> "DataFrame":
         └──────┴──────┘
 
         """
-        if subset is not None and isinstance(subset, str):
```

Comment:
This is not allowed according to the type annotation, nor tested, so I removed it.

Reply:
This is so that we can call both:

`df.drop_nulls("foo")`

And `df.drop_nulls(["foo"])`

Reply:
Okay, I was under the impression we would not want that anymore given there was no test and no type annotation for that case. I now added those.